scripts/install2

```diff
@@ -42,12 +46,7 @@ fi
 
 echo -e "\e[1;32mStarting installation on ${BOARD_FAMILY} ${BOARD_NAME} (${BOARD_REVISION})...\e[0m"
 
-<<<<<<< HEAD
-
-# Disable predictable network interface naming newer kernels
```

Comment:
?

Reply:
Merge errors I thought I committed the fix. Keeps creeping up from. Somewhere :#

Reply:
oh it was a bad commit, its already been corrected from a previous commit somewhere..